src/api/database/changes.rst

```diff
@@ -685,7 +685,7 @@ amount of duplicated code.
     While :ref:`map functions <mapfun>` doesn't process the design documents,
     using ``_view`` filter forces them to do this. You need to be sure, that
     they are ready to handle documents with *alien* structure without panic
-    crush.
+    crash.
```

Comment:
I suggest just removing the word "crash" entirely. "panic crash" seems redundant to me.